### PR TITLE
fix(DPLAN-1881 and DPLAN-1941): the bullet points are not being applied

### PR DESCRIPTION
**Pull request by sakutademos**
### Ticket
[DPLAN-1881]([url](https://demoseurope.youtrack.cloud/issue/DPLAN-1881/Fehlende-Formatierung-fur-Aufzahlungen-und-nummerierte-Listen-in-der-Uberprufungsansicht))

and [DPLAN-1941](https://demoseurope.youtrack.cloud/issue/DPLAN-1941/Aufzahlungen-werden-nicht-ubernommen)

**Description:** This PR fixes an issue with list styling by adding the .`c-styled-html` class to each part where the list style is needed.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

**Review by spiess-demos — CHANGES_REQUESTED** [review]
In other places we use the [.c-styled-html](https://github.com/demos-europe/demosplan-core/blob/b436e48166920d6dd82016a4c75d93d208770b12/demosplan/DemosPlanCoreBundle/Resources/client/scss/components/_styled-html.scss#L18) class. You could tweak that responsibly.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: templates/bundles/DemosPlanCoreBundle/DemosPlanFaq/faqlist.html.twig [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
         {% block content %}
 
-            <div id="scFaq" class="{{ 'layout__item u-1-of-1 u-3-of-4-lap-up u-pl-0'|prefixClass }}">
+            <div id="scFaq" class="{{ 'layout__item u-1-of-1 u-3-of-4-lap-up u-pl-0 c-styled-html'|prefixClass }}">
```
[the remark this answers is not in the document]
Comment (sakutademos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanFaq/faqlist.html.twig]: done here https://github.com/demos-europe/demosplan-core/pull/3576/commits/249bcdb2fc64cbd7ed78eb271a7e802004061797

File: templates/bundles/DemosPlanCoreBundle/DemosPlanNews/newsdetail.html.twig [hunk 2]
```diff
@@ -17,7 +17,7 @@
         {% endblock aside %}
 
         {% block content %}
-            <div class="u-3-of-4-desk-up">
+            <div class="u-3-of-4-desk-up c-styled-html">
                 <p><strong>{{ news.description|wysiwyg }}</strong></p>
                 <p>{{ news.text|wysiwyg }}</p>
```
[the remark this answers is not in the document]
Comment (sakutademos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanNews/newsdetail.html.twig]: done https://github.com/demos-europe/demosplan-core/pull/3576/commits/249bcdb2fc64cbd7ed78eb271a7e802004061797

File: templates/bundles/DemosPlanCoreBundle/DemosPlanNews/public_newsdetail.html.twig [hunk 3]
```diff
@@ -17,7 +17,7 @@
         hideMetaMenu: true
     } %}
 
-    <div class="layout--flush bg-color--white o-page__padded u-mb">
+    <div class="layout--flush bg-color--white o-page__padded u-mb c-styled-html">
```
[the remark this answers is not in the document]
Comment (sakutademos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanNews/public_newsdetail.html.twig]: done https://github.com/demos-europe/demosplan-core/pull/3576/commits/249bcdb2fc64cbd7ed78eb271a7e802004061797

**Review by spiess-demos — APPROVED** [review]
Lgtm!

**Pull request merged**